Source/macOS/LoopbackHTTPServer/OIDLoopbackHTTPServer.h

```diff
@@ -72,6 +72,9 @@ typedef enum {
 - (BOOL)start:(NSError **)error;
 - (BOOL)stop;
 
+- (BOOL)hasIPv4Socket;
```

Comment:
This comment was more to do with making life easy for consumers of this class - since the only reason consumers care about these values is in the context of determining which loopback address to use.